warehouse/models/mart/gtfs_schedule_latest/dim_gtfs_datasets_latest.sql

```diff
@@ -0,0 +1,30 @@
+WITH
+unfiltered_entries_latest AS (
+    {{ get_latest_schedule_data(
+    table_name = ref('dim_gtfs_datasets'),
+    clean_table_name = 'dim_gtfs_datasets'
+    ) }}
+),
```

Comment:
I don't think this will work, `get_latest_schedule_data` is for feed-versioned GTFS Schedule data but `dim_gtfs_datasets` is versioned based on its source data (i.e. Airtable) extracts.

You should be able to use just `where _is_current`; the model errors for me currently.

Reply:
I had done this locally, just neglected to push the latest change before marking ready for review. Fixed in [99c47ab](https://github.com/cal-itp/data-infra/pull/2434/commits/99c47ab9064a2750b2008d52206455efa4971ed4)